src/lib/pages/frontsite/PatronProfile/PatronShowLink.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Container, Grid, Icon, Divider } from 'semantic-ui-react';
+
+export const PatronShowLink = ({
+  items,
+  onShowAllClick,
+  onShowLessClick,
+  rowsPerPage,
+}) => {
+  const allHits = items.total === items.hits.length ? true : false;
+  return (
+    items.total > 0 &&
+    items.total > rowsPerPage && (
+      <Grid>
+        <Grid.Column textAlign="center">
+          {!allHits ? (
+            <>
+              <Divider hidden />
+              <Container className="link-like" onClick={() => onShowAllClick()}>
```

Comment:
```suggestion
              <a href="#" onClick={() => onShowAllClick()}>
```
you will probably avoid adding custom CSS in this way (to be checked)

Reply:
The drawback of this implementation (tested) was that it sends you to the top of the page (instead of the top of the refreshed component).

Reply:
thats probably the fault of `href="#" ` did you try other options? For example `href={javascript:void(0)}`?

Reply:
I have tested using javascript:void(0) in an "a" tag and it works as intended. Still, in the case of the Link element, I didn't manage to make it work as we want. 
I'm reluctant to use an "a" with a void href here instead of the current implementation. I see the point of trying to use Link, both to keep the use of elements consistent with the rest of the applications and to avoid overriding css. But I just don't see the point of using "a" here, because: we are not being consistent, we still need to use custom css to keep it as in the links of the frontpage, and we are using an hyperlink tag to something whose functionality is way more closer to a button than to a link (in fact, it doesn't involve any navigation at all).

Reply:
The only advantage of hyperlinks, at least the one I can think of, is accessibility: screen readers will "read" the cmp as a link: in this case, a `Container` is probably a `<div>` and a screen reader will probably not treat it as clickable.
Given said so, we did not put efforts in making InvenioILS accessible, but I would say that we can make efforts to add something that is not accessible.
Did you have a look [here](https://stackoverflow.com/questions/1631924/preventing-onclick-page-jumps), for example?